templates/pkg/crd_manager.go.tpl

```diff
@@ -5,6 +5,8 @@ package {{ .CRD.Names.Snake }}
 import (
 	"context"
 	"fmt"
+	ackerr "github.com/aws/aws-controllers-k8s/pkg/errors"
+    corev1 "k8s.io/api/core/v1"
```

Comment:
femto-nit, BTW: this is indented wrong (make sure you replace four spaces with a TAB character in templates :) )

Reply:
done